tests/h/auth/policy_test.py

```diff
@@ -372,6 +372,16 @@ def test_forget_does_not_proxy_to_client_if_path_mismatch(
         assert not client_policy.forget.call_count
         assert forgot == user_policy.forget.return_value
 
+    def test_forget_does_not_proxy_with_no_matched_route(
+        self, pyramid_request, api_policy, user_policy, client_policy
+    ):
+        pyramid_request.matched_route = None
+        user_policy.forget.return_value = []
+
+        forgot = api_policy.forget(pyramid_request)
+        assert not client_policy.forget.call_count
+        assert forgot == user_policy.forget.return_value
```

Comment:
This is crap test, but this whole test class isn't very good and will be rewritten soon enough.